src/ACMESharp/Protocol/AcmeProtocolClient.cs

```diff
@@ -615,6 +615,32 @@ private IJwsTool ResolveDefaultSigner()
             }
         }
 
+        /// <summary>
+        /// Revoke certificate
+        /// </summary>
+        /// <remarks>
+        /// https://tools.ietf.org/html/draft-ietf-acme-acme-18#section-7.6
+        /// </remarks>
+        public async Task<bool> RevokeCertificateAsync(
```

Comment:
On change I would like to propose is to change the return type.  Since the server doesn't return any value or content (as per the example in the spec), and any non-exception is considered successful, I don't think returning a `bool` adds any value.  So I suggest changing to `Task<bool>` to just `Task`.

Reply:
Agreed, I just didn't want it to be void, hadn't even considered the regular non-generic Task.